src/openforms/appointments/contrib/jcc/plugin.py

```diff
@@ -0,0 +1,183 @@
+import logging
+from datetime import date, datetime, timedelta
+
+from zeep import Client
+
+from ..base import (
+    AppointmentClient,
+    AppointmentLocation,
+    AppointmentProduct,
+    BasePlugin,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def squash_ids(lst):
+    return ",".join([i.identifier for i in lst])
+
+
+class Plugin(BasePlugin):
+    """
+    Plugin for JCC-Afspraken internetafsprakenadapter GGS2 (april 2020)
+    """
```

Comment:
Do we have any documentation links we can include here?

Reply:
Unfortunately, no. I had to move heaven and earth to get this sent to me as PDF by one of the municipalities.